src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -228,18 +230,36 @@ class adminLogs extends viewModelBase {
         return item.includes("EXCEPTION:") || item.includes("Exception:") || item.includes("FATAL ERROR:");
     }
     
-    private isCustomizedItem(item: string): boolean {
-        return (item.includes("Connection established") || item.includes("Connection closed") || item.includes("Connection paused"));
+    private isStudioItem(item: string): boolean {
+        return item.includes(adminLogs.studioMsgPart);
     }
     
+    private getAddedClass(item: string) {
+        const isStudioItem = this.isStudioItem(item);
+        
+        if (this.hasError(item)) {
+            return "bg-danger";
+        }
+        
+        if (isStudioItem && item.includes("Connection interrupted by server")) {
+            return "text-danger";
+        }
+        
+        if (isStudioItem) {
+            return "studio-item-info";
```

Comment:
can we use text-info ?

Reply:
We can... 
but I find it so much nicer with: `darken(@state-info-text, 7%);` in the less file
It really looks much better on the dark background